test/e2e/maintenance-event-dry-run-test

```diff
@@ -101,6 +101,7 @@ for i in $(seq 0 10); do
       echo "✅ Scheduled Maintenance Event Dry Run Test Passed $CLUSTER_NAME! ✅"
       exit 0
   fi
+    echo "Assertion Loop $i/10, sleeping for 10 seconds"
```

Comment:
do you think it's worth going with the larger sleep value to cover all tests and be consistent?

Reply:
yeah, this didn't even make any sense. I was being lazy, yet doing so, led to more work and now rework 🤷‍♂️